ocw/lib/provider.py

```diff
@@ -3,7 +3,7 @@
 from datetime import datetime
 from datetime import timedelta
 from datetime import timezone
-from distutils.version import LooseVersion
+from looseversion import LooseVersion
```

Comment:
this is super awesome ! I was expecting a lot of work with this and it appears just one-liner , cool !

Reply:
unfortuqanelty new dep :(